src/domain/Tables/Table/subcomponents/TableRow.tsx

```diff
@@ -55,6 +56,7 @@ interface IUseDragProps {
 }
 
 interface IGetLeftCell <T extends {}> {
+  t: any
```

Comment:
it kills me that I keep seeing `any` but I'll allow these for the sake of this being urgent lol